src/lib/AddressUpgradeable.sol

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.2 <0.8.0;
+
+/**
+ * @dev Collection of functions related to the address type
+ */
+library AddressUpgradeable {
```

Comment:
Is there a reason why all the open zepelling libraries and contracts are not pulled from OZ import directly?